package.json

```diff
@@ -9,10 +9,10 @@
     "lint": "next lint",
     "test": "jest tests/unit",
     "lint:fix": "eslint --fix */**/*.js tests/**/*.js",
-    "cy:open": "cypress open --env RELEASE_VERSION=full",
-    "cy:open:mvp": "cypress open --env RELEASE_VERSION=mvp",
-    "cy:run": "cypress run --env RELEASE_VERSION=full",
-    "cy:run:mvp": "cypress run --env RELEASE_VERSION=mvp  --spec 'tests/cypress/integration/reportRepair/*Mvp.spec.js'",
+    "cy:open": "cypress open --env RELEASE_VERSION=full --config baseUrl=http://localhost:3000",
+    "cy:open:mvp": "cypress open --env RELEASE_VERSION=mvp --config baseUrl=http://localhost:3000,testFiles=**/*Mvp.spec.js",
+    "cy:run": "cypress run --env RELEASE_VERSION=full --config baseUrl=http://localhost:3000",
+    "cy:run:mvp": "cypress run --env RELEASE_VERSION=mvp --config baseUrl=http://localhost:3000 --spec '**/*Mvp.spec.js'",
```

Comment:
Updated `spec` to include all `*Mvp.spec.js` files as the previous value was not including any files outside `reportRepair` folder, i.e. `landingPageMvp.spec.js`

Reply:
Tests failing. Is it because you didn't check in the baseURL as part of the cypress config file?

Reply:
Yes, this wasn't included in the CI definition 🤦 - have updated now.